### DISCUSSION TITLE
Wordkeys request

### DISCUSSION THREAD

**Discussion by gitcodehelp** [sheltermanager/asm3#1493] [Q&A / Help]
Could you please implement wordkeys for:

BroughtInByIDNumber
ReturnedByIDNumber
ReturnedBy Additional Fields

We would like to include Driver's License numbers on our intake paperwork -- thank you.

**Comment by bobintetley** [comment]
Hi, why would you need ID for the person who brought an animal in? In the case of ReturnedBy, it is most likely to be the adopter so it is already available as OwnerIDNumber. In the case of BroughtInBy, the ID of the original owner is available.

I will add returnedby additional fields to our dev list.

**Reply by gitcodehelp** [reply]
Our shelter (like many) requires people who bring us animals to show proof of ID and sign a contract affirming the surrender circumstances. Having their ID number on the contract helps to uniquely identify that individual for legal purposes, because
(sadly) we encounter these situations:

1) After a relationship split, ex-partners will attempt to relinquish their ex-lover's animal to us as "stray" for spite.

2) A family member who thinks their parent/sibling should not have an animal will surrender it to us claiming it is theirs. 

3) People will steal nuisance or neglected animals from their neighbors and claim they found it as "stray".

4) Being one of the few no-kill shelters in our area, we are often overcrowded and sometimes have to restrict where we take animals from. People lie and abuse this policy.  Requiring proof of ID makes people less likely to lie, and allows us to identify bad actors in our system.

Hence, wordkeys needed for:

BroughtInByIDNumber
ReturnedByIDNumber

Prior to the addition of the ID Number field, I was implementing it as an Additional Field, along with additionals for the ID Type (Driver's License, Passport, etc.) and ID State/Country -- hence the "ReturnedBy Additional Fields" request.

Thanks!

**Comment by bobintetley** [comment]
Ok, thanks for the reply, I'll put this on our dev list.
